src/policy/policy.cpp

```diff
@@ -53,7 +53,7 @@ CAmount GetDustThreshold(const CTxOut& txout, const CFeeRate& dustRelayFeeIn)
         nSize += (32 + 4 + 1 + 107 + 4); // the 148 mentioned above
     }
 
-    return dustRelayFeeIn.GetFee(nSize);
+    return dustRelayFeeIn.GetFee(nSize, txout.nAsset.GetAsset());
```

Comment:
Actually, this shouldn't be using the exchange rates at all.

If there's a transaction with an output of some asset that the node doesn't have an exchange rate for, then there's no meaningful conversion from the fee rate to an amount using exchange rates. So we should probably just treat `dustrelayfee` as a lower bound on the amount that doesn't have anything to do with the RFU. 

Otherwise, the node will reject outputs as dust just because it doesn't care about the asset, which is obviously wrong.

Reply:
Did some more investigating and realized that dust checks are currently limited to outputs that have the same asset as the fee, so there's no risk of different assets being compared.

This applies to in policy: https://github.com/ElementsProject/elements/blob/master/src/policy/policy.cpp#L149

And to wallet: https://github.com/ElementsProject/elements/blob/master/src/wallet/spend.cpp#L1143 and https://github.com/ElementsProject/elements/blob/master/src/wallet/spend.cpp#L1559